### ISSUE TITLE
Replace entries with non-GitHub links to their GitHub repository

### ISSUE THREAD

**Issue by ResamVi** [ResamVi/awesome-java-stars#1]
Right now, any entry that does not have [a "github.com" link gets skipped](https://github.com/ResamVi/awesome-java-stars/blob/master/src/parser/Parser.java#L67). 

There are some projects that do have their code on GitHub though which should be replaced.

Example: https://github.com/ResamVi/awesome-java-stars/pull/2

_Please make single PRs for each entry._

**Comment by parthnagdev** [comment]
Can I do a single PR for all the replacements? because I think hacktoberfest has a rule that you cannot just correct a single line of code and call it a PR, it won't count.

**Comment by ResamVi** [comment]
I wasn't aware of that, go ahead!

**Comment by parthnagdev** [comment]
Hey I was trying to push a commit on a new branch(git push origin more-links) I created but I got this error:

remote: Permission to ResamVi/awesome-java-stars.git denied to parthnagdev.
fatal: unable to access 'https://github.com/ResamVi/awesome-java-stars.git/': The requested URL returned error: 403

Sorry for being a hassle this is my first time doing a commit. 

Edit: ok I understand I need to Fork the repo.

**Comment by ResamVi** [comment]
> Edit: ok I understand I need to Fork the repo.

Yup! You will have to do a commit to your fork and push those changes.

Afterwards you can navigate to your fork and create a Pull Request from the UI

![image](https://github.com/ResamVi/awesome-java-stars/assets/6261556/c67da865-063f-42df-912d-1359e6a42991)
